### ISSUE TITLE
nux dextop package

### ISSUE THREAD

**Issue by hairmare** [radiorabe/centos-rpm-ffmpeg#1]
Nux dextop already has an ffmpeg package `ffmpeg-2.6.8`. Is there a reason we can't use that?

**Comment by paraenggu** [comment]
Yes, I need the 3.1 branch (which includes lots of fixes) and I also want to have a build without all the video related dependency as I only need audio stuff.

**Comment by hairmare** [comment]
You may want to try requesting a bump in the [forums](https://forums.nux.ro/index.php?t=thread&frm_id=10&S=832b5ea1f4555c1a1ee609574e3e51c4) before forking the whole shebang over here.
